template-for-repository/proofs/Makefile.common

```diff
@@ -158,6 +159,8 @@ CHECKFLAGS += $(CBMC_FLAG_SIGNED_OVERFLOW_CHECK)
 CHECKFLAGS += $(CBMC_FLAG_UNDEFINED_SHIFT_CHECK)
 CHECKFLAGS += $(CBMC_FLAG_UNSIGNED_OVERFLOW_CHECK)
 
+CHECKFLAGS += --unwinding-assertions
```

Comment:
Why not use the same defined-variable approach here?  Yes, everyone should use it, but we want to encourage best practice without forcing anything.

Reply:
done